packages/frontend/src/layout/sideColumn/sideColumn.tsx

```diff
@@ -13,15 +13,23 @@ import { Page } from '../../constants'
 const SideColumn = () => {
     const history = useHistory()
     const userContext = useContext(UserContext)
-
     const page = window.location.pathname as any
+    const [showBackBtn, setShowBackBtn] = useState<boolean>(false)
 
     const gotoSetting = () => {
         if (userContext.userState) {
             history.push('/setting', { isConfirmed: true })
         }
     }
 
+    window.onscroll = () => {
```

Comment:
We should make a new context for UI flags and put this handler in that context. Then the `window.onscroll` function can update an observable variable so that many components can watch the scroll position without registering many handlers.

Reply:
This can be the same context as in #70

Reply:
already new a UI context in #70, after merging that branch, then this one can be modified based on that